espanol/rezando/paquete/NOSE/ayuntamiento/ayuntamiento

```diff
@@ -0,0 +1 @@
+Una vez allí, y después de pagar las tasas por el informe, pudo obtener la dirección de Ester
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.